spring-cloud-starter-stream-source-twitterstream/src/test/java/org/springframework/cloud/stream/app/twitterstream/source/TwitterStreamMessageProducerTest.java

```diff
@@ -0,0 +1,45 @@
+package org.springframework.cloud.stream.app.twitterstream.source;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.social.twitter.api.impl.TwitterTemplate;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import static org.junit.Assert.*;
+
+public class TwitterStreamMessageProducerTest {
```

Comment:
The convention is `Tests` - plural. Even if there is only one `@Test` method in the class.